scripts/Run-CodeCoverage.ps1

```diff
@@ -20,18 +20,17 @@ $nunitDirectory = Join-Path $rootDirectory packages\NUnit.ConsoleRunner.3.7.0\to
 $nunitConsoleRunner = Join-Path $nunitDirectory nunit3-console.exe
 
 $testAssemblies = @(
-    "test\GitHub.Api.UnitTests\bin\$Configuration\GitHub.Api.UnitTests.dll",
-    "test\GitHub.App.UnitTests\bin\$Configuration\GitHub.App.UnitTests.dll",
-    "test\GitHub.Exports.Reactive.UnitTests\bin\$Configuration\GitHub.Exports.Reactive.UnitTests.dll",
-    "test\GitHub.Exports.UnitTests\bin\$Configuration\GitHub.Exports.UnitTests.dll",
-    "test\GitHub.Extensions.UnitTests\bin\$Configuration\GitHub.Extensions.UnitTests.dll",
-    "test\GitHub.InlineReviews.UnitTests\bin\$Configuration\GitHub.InlineReviews.UnitTests.dll",
-    "test\GitHub.Primitives.UnitTests\bin\$Configuration\GitHub.Primitives.UnitTests.dll",
-    "test\GitHub.TeamFoundation.UnitTests\bin\$Configuration\GitHub.TeamFoundation.UnitTests.dll",
-    "test\GitHub.UI.UnitTests\bin\$Configuration\GitHub.UI.UnitTests.dll",
-    "test\GitHub.VisualStudio.UnitTests\bin\$Configuration\GitHub.VisualStudio.UnitTests.dll",
+    "test\GitHub.Api.UnitTests\bin\$Configuration\net461\GitHub.Api.UnitTests.dll",
```

Comment:
This is likely to change to `net46` at some point. I wonder if this will accept wildcards?

Reply:
I don't think it will take wildcards at this point